wcfsetup/install/files/lib/system/moderation/queue/AbstractCommentResponseModerationQueueHandler.class.php

```diff
@@ -163,4 +164,26 @@ public function removeContent(ModerationQueue $queue, $message)
             $responseAction->executeAction();
         }
     }
+
+    #[\Override]
+    public function isAffectedUser(ModerationQueue $queue, $userID)
+    {
+        if (!AbstractModerationQueueHandler::isAffectedUser($queue, $userID)) {
```

Comment:
```suggestion
        if (!parent::isAffectedUser($queue, $userID)) {
```

Reply:
I cannot use this because `AbstractCommentResponseModerationQueueHandler` extends `AbstractCommentCommentModerationQueueHandler` and in the function `isAffectedUser` `$queue->objectID` is not a comment id it is the comment response id.